cognite_toolkit/_cdf_tk/commands/auth.py

```diff
@@ -363,20 +364,25 @@ def check_auth(
         if not has_function_read_access:
             self.warn(HighSeverityWarning("Cannot check function service status, missing function read access."))
             return None
-        function_status = ToolGlobals.client.functions.status()
-        if function_status.status != "activated":
-            if function_status.status == "requested":
-                print("  [bold yellow]INFO:[/] Function service activation is in progress (may take up to 2 hours)...")
-            else:
-                if not dry_run:
+        try:
+            function_status = ToolGlobals.client.functions.status()
```

Comment:
Too broad exception clause
```suggestion
        try:
            function_status = ToolGlobals.client.functions.status()
        except CogniteAPIError as e:
            self.warn(HighSeverityWarning(f"Unable to check function service status.\n{e}"))
```